### PR TITLE
[Expression]Add abs and sqrt built- in function

### LINKED ISSUE #5114: add abs and float.Epsilon in adaptive expression
### Use this [query](https://github.com/Microsoft/botbuilder-dotnet/issues?q=is%3Aissue+is%3Aopen++label%3Afeature-request+) to search for the most popular feature requests.

**Is your feature request related to a problem? Please describe.**

When submitting this pr https://github.com/microsoft/BotFramework-Composer/pull/5575/files, I found we lack these two in adaptive expression

**Describe the solution you'd like**

Add them like C# https://docs.microsoft.com/en-us/dotnet/api/system.math.abs?view=net-5.0, https://docs.microsoft.com/en-us/dotnet/api/system.double.epsilon?view=net-5.0

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by Danieladu**
Fixes #5114

## Description
Add `abs` and `sqrt` prebuilt function.
Example:
```
abs(3.12134)  -> 3.12134
abs(-3.12134) -> 3.12134
abs(0) -> 0
sqrt(9) -> 3
sqrt(-1)  -> throw error
sqrt(0) ->  0
```

### CODE REVIEW

**Comment by mrivera-ms** [comment]
@luhan2017, could you please review this?

File: libraries/AdaptiveExpressions/BuiltinFunctions/Sqrt.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        {
+            return FunctionUtils.ApplyWithError(
+                        args =>
+                        {
+                            string error = null;
+                            object result = null;
+                            var originalNumber = Convert.ToDouble(args[0], CultureInfo.InvariantCulture);
+                            if (originalNumber < 0)
+                            {
+                                error = "Do not support square root extraction of negative numbers currently.";
```
Comment (mrivera-ms) [inline libraries/AdaptiveExpressions/BuiltinFunctions/Sqrt.cs:32]: Remove "currently" from the error message.

**Review by mrivera-ms — CHANGES_REQUESTED** [review]

File: libraries/AdaptiveExpressions/BuiltinFunctions/Sqrt.cs (hunk 1 above)
Comment (Danieladu) [reply libraries/AdaptiveExpressions/BuiltinFunctions/Sqrt.cs:32]: Ok. The orginal reason for 'current' is that currently, SDK does not support the square root of negative number in Real number scope. But in the future, we may consider supporting Complex number, after that the square root of the negative number is possible.
Anyway, remove the `currently` is acceptable.

**Review by mrivera-ms — APPROVED** [review]

**Comment by boydc2014** [comment]
LGTM

**Pull request merged**